google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -810,6 +810,27 @@ public long getCurrentUploadOffset(String uploadId) {
     }
   }
 
+  @Override
+  public StorageObject queryResumableUpload(String uploadId, long totalBytes) {
```

Comment:
Could use a docstring. What is this used for and when during the upload should it be called? What does totalBytes represent here?

Reply:
+1